frontend/validator.js

```diff
@@ -34,14 +35,18 @@ var Validator = (function() {
     req.send(JSON.stringify({ schema: content, }));
   }
 
+  /**
+   * Fetch from remote SpaceAPI endpoint
+   */
   function validate_url(ev) {
     banner.className = 'fetching';
     ev.preventDefault();
     offlineCheck();
 
     var url = getValue(url_input);
     if (url.indexOf('http://') === -1 && url.indexOf('https://') === -1) {
-      appendLog('\nInvalid URL: '+url);
+      appendLog('Invalid URL: '+url);
+      setBanner('ERROR', 'Invalid URL: '+url);
```

Comment:
I don't think we should validate URLs by testing for the presence of "https?". Why not send a request and then see if that fails because of an invalid URL? Or is there no API for that?